### DISCUSSION TITLE
关于分支管理讨论

### DISCUSSION THREAD

**Discussion by JeffersonHuang** [h65wang/flutter_interview_questions#69] [General]
## 已完成
* 创建了release。并将web deploy actions 的触发条件改为了release 的push。
* 删除了main 分支中的flutter_web_deploy.yaml 文件。只保留了flutter_chekc.yaml文件，用于检查格式。
## PR流程(待讨论)
* 贡献者这提交pr 到 main，由 @h65wang 合并到release分支？
* release是否需要限制权限？

**Comment by JeffersonHuang** [comment]
@h65wang @shirne @zhuzhao404 @huanghui1998hhh @WeiJun0507 。大佬们，有啥好的想法吗？

**Reply by WeiJun0507** [reply]
这个安排挺好的😄
